amp/repository/categorymanager/view/createOrEditCategory.jsp

```diff
@@ -1,509 +1,514 @@
+</c:set>
+
+<c:set var="translation2">
+	<digi:trn key="aim:categoryManagerPlsEnterKey">You need to enter a key for the category</digi:trn>
+</c:set>
+
+<c:set var="translation3">
+	<digi:trn key="aim:categoryManagerMoreThan2Values">Category must have at least two possible values</digi:trn>
+</c:set>
+
+<c:set var="translation4">
+	<digi:trn key="aim:CategoryOnlyOneCountryValue">Category must have one country value</digi:trn>
+</c:set>
+
+<c:set var="translation5">
+	<digi:trn key="aim:CategoryOnlyOneRegionValue">Category must have one region value</digi:trn>
+</c:set>
+<c:set var="translation6">
+	<digi:trn key="cm:correctNumberOfFields">Please specify a number of fields to add that is above zero and below 30</digi:trn>
+</c:set>
+<c:set var="translation7">
+	<digi:trn key="cm:warningDeleteCategoryValue">Please proceed only if you are sure that the value you are trying to delete is not used by the system anymore.  Are you sure you want to delete it ?</digi:trn>
+</c:set>
+
+<c:set var="translation8">
+	<digi:trn key="cm:warningDeleteLabelCategory">Are you sure you want to remove this label category ?</digi:trn>
+</c:set>
+<c:set var="translation9">
+	<digi:trn>Title cannot be empty and can only contain letters, digits, _, /, (), #, -, : and space. </digi:trn>
+</c:set>
+
+<c:set var="translation10">
+	<digi:trn>Please enter key for category value</digi:trn>
+</c:set>
+
+<script type="text/javascript">
+	labelPanels					= new Array();
+    var isRtl = <%=TLSUtils.getCurrentLocale().getLeftToRight() == false%>;
+    var language = '<%=TLSUtils.getCurrentLocale().getCode()%>';
+    var region = '<%=TLSUtils.getCurrentLocale().getRegion()%>';
+	
+	function addLabelCategory() {
+		var subForm				= document.forms["cmCategoryManagerForm"];
+		subForm.action			= "/categorymanager/categoryLabelsAction.do";
+		subForm.useAction.value	= "addCategory";
+		subForm.submit();
+	}
+	function addLabelValues(possibleValueIndex, labelCategoryIndex) {
+		var subForm				= document.forms["cmCategoryManagerForm"];
+		subForm.action			= "/categorymanager/categoryLabelsAction.do?possibleValueIndex=" + possibleValueIndex + 
+										"&labelCategoryIndex=" + labelCategoryIndex;
+		subForm.useAction.value	= "addLabelValue";
+		subForm.submit();
+	}
+	function delLabelCategory(categoryId) {
+		if ( confirm('${translation8}') ) {
+			var subForm				= document.forms["cmCategoryManagerForm"];
+			subForm.action			= "/categorymanager/categoryLabelsAction.do";
+			subForm.useAction.value	= "delCategory";
+			subForm.delUsedCategoryId.value	= categoryId;
+			//alert(subForm.delUsedCategoryId.value);
+			subForm.submit();
+		}
+		return false;
+	}
+      
+    function addNewValue(position) {
+    	var subForm				= document.forms["cmCategoryManagerForm"];
+    	var fieldNumStr			= subForm.numOfAdditionalFields.value;
+    	var fieldNum			= parseInt(TranslationManager.convertNumbersToEasternArabicIfNeeded(isRtl, language, region, "" + fieldNumStr));
```

Comment:
You also need to convert to western if needed cause a eastern number will evaluate to NaN